sdk/monitor/monitor-query/README.md

```diff
@@ -440,6 +459,7 @@ QueryMetricsResult
     |---timeseries (list of `TimeSeriesElement` objects)
         |---metadataValues
         |---data (list of data points represented by `MetricValue` objects)
+|---getMetricByName(metricName): Metric (convenience method)
```

Comment:
We should also have a sample somewhere for this new function.

Reply:
I have added these two lines to the existing sample - 
```
const metric = metricsResponse.getMetricByName(firstMetric.name!);
console.log(`Selected Metric: ${firstMetric.name}`, JSON.stringify(metric, undefined, 2));
```